src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -0,0 +1,72 @@
+package io.iohk.ethereum.domain
+
+import java.math.BigInteger
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.crypto.ECDSASignature
+import io.iohk.ethereum.rlp.{encode => rlpEncode, _}
+import io.iohk.ethereum.rlp.RLPImplicits._
+import org.spongycastle.util.encoders.Hex
+
+
+object SignedTransaction {
+
+  val FirstByteOfAddress = 12
+  val AddressLength = 20
+  val LastByteOfAddress = FirstByteOfAddress + AddressLength
+
+}
+
+case class SignedTransaction(tx: Transaction,
+                             pointSign: Byte, //v - 27 or 28 according to yellow paper, but it is 37 and 38 in ETH
+                             signatureRandom: ByteString, //r
+                             signature: ByteString /*s*/) {
+
+  import SignedTransaction._
+
+  lazy val bytesToSign: Array[Byte] = crypto.sha3(
+    rlpEncode(RLPList(tx.nonce, tx.gasPrice, tx.gasLimit,
+                      tx.receivingAddress.toArray,
+                      tx.value,
+                      tx.payload)))
```

Comment:
Similarly as with declarations, for multi-line invocations it's best to put arguments on their own lines to avoid future re-indents: http://docs.scala-lang.org/style/indentation#methods-with-numerous-arguments